env-example.toml

```diff
@@ -5,8 +5,6 @@
 # which may be used by several components.
 #
 ["aws"]
-access_key_id = "<aws access key id>"
-secret_access_key = "<aws secret access key>"
 region = "<aws region>"
```

Comment:
It's odd that this TOML section now only has the region. Don't we specify it as a env var as well?

Reply:
It is indeed odd. I am not sure why the current version of the code is not picking it up automatically. We have to investigate, and maybe get rid of this altogether.